index.html

```diff
@@ -0,0 +1,203 @@
+                </p>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-gitHub.png" alt="Github Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-linkedin.png" alt="Linked-in  Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-angelist.png" alt="Github Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-twitter.png" alt="Twitter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-medium.png" alt="Github Page">
+                        </a>
+                    </li>
+                </ul>
+            </section>
+            <section class="projects-grid">
+                <div class="grid-title">
+                    <h3>My Recent Works</h3>   
+                </div>
+                <article class="featured">
+                    <img src="/assets/project-featured.jpg" alt="Menu Button">                        
+                    <div class="wrap-card">
+                        <h3 id="article-h">Multi-Post Stories</h3>
+                        <p id="featured-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        <div class="lang">
+                            <button type="button">css</button>
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div>
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article>
+                <article class="project1">
+                    <div class="wrap-card">
+                        <h3>Profesional Art Printing Data</h3>
+                        <p>A daily selection of privately personalized reads;  no accounts or sign-ups required. has been the industry's standard</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
```

Comment:
- [ ] [REQUIRED] The list of project technologies should(e.g., "JavasScript", "HTML" and "CSS") a list `ul` and each of them a `li`. Please make changes!